api/base/filters.py

```diff
@@ -61,6 +63,78 @@ def filter_queryset(self, request, queryset, view):
         return queryset
 
 
+class MultiValueCharFilter(django_filters.BaseInFilter, django_filters.filters.CharFilter):
+
+    def filter(self, qs, value):
+        q = Q()
+        values = value or []
+        for value in values:
+            name = self.name
+            q |= Q(**{'{}__icontains'.format(name): value})
+        qs = qs.filter(q)
+
+        return qs
+
+
+class JSONAPIFilterSet(django_filters.FilterSet):
+
+    QUERY_PATTERN = re.compile(r'^filter\[(?P<fields>((?:,*\s*\w+)*))\](\[(?P<op>\w+)\])?$')
```

Comment:
Not yet! This was used when parsing out the other operators used in other filters, such as `eq`, `ne`, etc, which aren't used in this endpoint that I picked. Perhaps this proof of concept should include porting a second endpoint that uses operators?